src/main/java/jewellery/inventory/dto/response/resource/ResourceInUserResponseDto.java

```diff
@@ -1,12 +1,12 @@
 package jewellery.inventory.dto.response.resource;
 
-import java.util.UUID;
+import java.util.List;
+import jewellery.inventory.dto.response.UserResponseDto;
 import lombok.Data;
 
 @Data
 public class ResourceInUserResponseDto {
-  private UUID resourceId;
-  private String ownerName;
-  private String resourceClazz;
+  private UserResponseDto owner;
+  List<ResourceResponseDto> resources;
```

Comment:
The list should be a list of (resource, quantity) <=> List<Pair<ResourceResponseDto, Double>>

Reply:
ResourceInUserRequest should remain as it is.

Reply:
What is the scenario in which we are going to use this response? The list of resource dtos can't map to a single quantity field.

Reply:
At the time you wrote your comment I had not push the new code yet. The new DTO is not a response DTO. It is used in the ResourcesInUserResponseDto. The new version is
`@Data
public class ResourcesInUserResponseDto {
  private UserResponseDto owner;
  private List<ResourceQuantityDto> resources;
}
`
And the new DTO looks like this:
`@Data
public class ResourceQuantityDto {
    private ResourceResponseDto resource;
    private double quantity;
}`
Maybe the naming ResourceQuantityDto  is not the best

Reply:
yep, this looks better. I don't mind the name. Maybe more appropriate would be ResourceAndQuantityDto